src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -116,11 +132,63 @@ public static int getHearingDuration(SscsCaseData caseData) {
                 return Integer.parseInt(caseData.getAdjournCaseNextHearingListingDuration()) * DURATION_HOURS_MULTIPLIER;
             }
         }
-        if (nonNull(caseData.getBenefitCode()) && nonNull(caseData.getIssueCode())) {
-            // TODO Dependant on SSCS-10116 - Will use Session Category Reference Data
-            return 45;
+
+        return null;
+    }
+
+    public static Integer getHearingDurationBenefitIssueCodes(SscsCaseData caseData, ReferenceData referenceData) {
+        HearingDuration hearingDuration = referenceData.getHearingDurations().getHearingDuration(
+            caseData.getBenefitCode(), caseData.getIssueCode());
+
+        if (nonNull(hearingDuration)) {
+
+            if (isYes(caseData.getAppeal().getHearingOptions().getWantsToAttend())) {
+                Integer duration = HearingsCaseMapping.isInterpreterRequired(caseData)
+                    ? hearingDuration.getDurationInterpreter()
+                    : hearingDuration.getDurationFaceToFace();
+                return referenceData.getHearingDurations().addExtraTimeIfNeeded(duration, hearingDuration.getBenefitCode(), hearingDuration.getIssue(), getElementsDisputed(caseData));
+            } else if (PAPER.getValue().equalsIgnoreCase(caseData.getAppeal().getHearingType())) {
+                return hearingDuration.getDurationPaper();
+            }
         }
```

Comment:
Better to jump out fast.  I.e.
if (!nonNull(hearingDuration)) {
    return null;
}
This also avoids nested if's too